src/file-manager/fm-desktop-icon-view.c

```diff
@@ -510,6 +553,13 @@ delayed_init (FMDesktopIconView *desktop_icon_view)
                              "done_loading",
                              G_CALLBACK (done_loading), desktop_icon_view, 0);
 
+    /*Handle the middle click on x11 */
+    if (GDK_IS_X11_DISPLAY (gdk_display_get_default))
```

Comment:
This is dangerously wrong :)  And breaks the X11 feature I guess
```suggestion
    if (GDK_IS_X11_DISPLAY (gdk_display_get_default ()))
```

Reply:
That was a cut and paste from elsewhere, maybe you just found my problem running this earlier

Reply:
YES! That was it-now I can move this back to it's original place